src/__test__/Missions.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../redux/store';
+import Missions from '../routes/Missions';
+
+describe('Mission test', () => {
+  test('to render all missions: ', () => {
+    const missions = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Missions />
+        </BrowserRouter>
+      </Provider>,
+    );
+    expect(missions).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] as part of the requirements you should  **_add tests for all your components and functions._** currently you tested only two components , kindly tests all of your components and add unit tests for your functions 🙏

Reply:
- Done for components Rockets, MyRockets, Layout and Navbar
- Done for all extra routes

@Talha-Hanif5666 Missions is pending